common/src/main/java/com/aliyun/emr/rss/common/network/server/MemoryTracker.java

```diff
@@ -115,8 +114,10 @@ private MemoryTracker(
         MemoryTrackerStat lastAction = memoryTrackerStat;
         memoryTrackerStat = currentMemoryAction();
         if (lastAction != memoryTrackerStat) {
+          logger.info("MemoryTrackerStat changed {} -> {}", lastAction, memoryTrackerStat);
```

Comment:
Should we use debug level?

Reply:
State change won't be a lot, so I think the info would be ok. If changed to debug, this log would be unnoticeable because the log of method "currentMemoryAction"  is huge.